app/views/aspect_memberships/_aspect_membership_dropdown.haml

```diff
@@ -14,18 +14,17 @@
       %li.aspect_selector{ :class => ('selected' if aspect_membership_ids[aspect.id].present?), 'data-aspect_id' => aspect.id, 'data-membership_id' => aspect_membership_ids[aspect.id], :tabindex => '0' }
         %a
           %span.status_indicator
-            %i.icon-ok
+            %i.glyphicon.glyphicon-ok
             %i.icon-refresh
           %span.text
             = aspect.name
 
-    - if (dropdown_may_create_new_aspect && defined?(person) && person)
+    - if dropdown_may_create_new_aspect && defined?(person) && person
       %li.divider
-      %li.newItem
-        .add_aspect
-          %a{ href: "#" }
-            = t("contacts.index.add_a_new_aspect")
+      %li.newItem.add_aspect
+        %a{ href: "#", :data => { :toggle => "modal", :target => "#newAspectModal" }}
```

Comment:
Hound doesn't complain in HAML views, but it would be awesome if you could update *all* instances of hashes to the Ruby 1.9 hash syntax where you touched them.